a10_neutron_lbaas/v1/handler_member.py

```diff
@@ -35,14 +36,17 @@ def _create(self, c, context, member):
         server_ip = self.neutron.member_get_ip(context, member,
                                                c.device_cfg['use_float'])
         server_name = self._meta_name(member, server_ip)
-
+        conn_limit = c.device_cfg['conn-limit']
```

Comment:
@Cedev Look 2 lines above where we do the same thing.  My only concern here is if we change the default-"fill in" behavior, these break spectacularly.